<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>powder_layout</class>
 <widget class="QWidget" name="powder_layout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>507</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_ps">
     <item row="4" column="1">
      <spacer name="verticalSpacer_ps">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="l_ps_end">
       <property name="text">
        <string>end</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="l_ps_start">
       <property name="text">
        <string>start</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="l_ps4">
       <property name="text">
        <string>det_rot</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="dSB_powder_start">
       <property name="minimum">
        <double>-20.000000000000000</double>
       </property>
       <property name="maximum">
        <double>-5.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QDoubleSpinBox" name="dSB_powder_end">
       <property name="minimum">
        <double>-20.000000000000000</double>
       </property>
       <property name="maximum">
        <double>-5.000000000000000</double>
       </property>
       <property name="value">
        <double>-10.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="4">
      <widget class="QPushButton" name="pB_ps">
       <property name="text">
        <string>Simulate</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="l_ps_shift">
       <property name="text">
        <string>shift</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="dSB_shift"/>
     </item>
     <item row="2" column="2">
      <widget class="QCheckBox" name="cB_labels">
       <property name="text">
        <string>labels</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="powd_plot_layout"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
